ai/yolo/auto_labeling_script/main.py

```
@@ -0,0 +1,206 @@
+import os
+import cv2
+import numpy as np
+
+
+def preprocess(img):
+
+    gray = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
+    clahe = cv2.createCLAHE(clipLimit=2.0, tileGridSize=(8, 8))
+    gray = clahe.apply(gray)
+    return gray
```

Ruff (0.15.9)

[warning] Unnecessary assignment to `gray` before `return` statement

Remove unnecessary assignment

(RET504)